bot/src/main/java/com/almightyalpaca/discord/jdabutler/commands/commands/DateVersionCommand.java

```diff
@@ -0,0 +1,124 @@
+package com.almightyalpaca.discord.jdabutler.commands.commands;
+
+import com.almightyalpaca.discord.jdabutler.Bot;
+import com.almightyalpaca.discord.jdabutler.commands.Command;
+import com.almightyalpaca.discord.jdabutler.util.EmbedUtil;
+import com.kantenkugel.discordbot.jenkinsutil.JenkinsApi;
+import com.kantenkugel.discordbot.jenkinsutil.JenkinsBuild;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
+
+import java.awt.Color;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+public class DateVersionCommand extends Command
+{
+    private static final String DATE_FORMAT = "yyyy-MM-dd; HH:mm:ss";
+    private static final String[] ALIASES = { "date", "published" };
+    private static final JenkinsApi JENKINS = JenkinsApi.JDA_JENKINS;
+    private static final Calendar CAL = Calendar.getInstance();
+    private static final DateFormat DFM = getDateFormat();
+
+    private static DateFormat getDateFormat() {
+        DateFormat dfm;
+        try
+        {
+            dfm = new SimpleDateFormat(DATE_FORMAT);
+        }
+        catch (NullPointerException | IllegalArgumentException ex)
+        {
+            final String defaultFormat = "dd.MM.yyyy; HH:mm:ss";
+            Bot.LOG.warn("Given format for DateVersionCommand was not valid, using: " + defaultFormat);
+            dfm = new SimpleDateFormat(defaultFormat);
+        }
+        return dfm;
+    }
+
+    @Override
+    public void dispatch(final User sender, final TextChannel channel, final Message message, final String content, final GuildMessageReceivedEvent event)
+    {
+        final EmbedBuilder eb = new EmbedBuilder();
```

Comment:
This can be moved down to where its actually used to make it easier to find

Reply:
Well, it needs to be outside of the try-catch block (because it's in the catch as well). Should I create a second builder then?

Reply:
Oh right. Didn't see that you used it in the catch block too

Reply:
Currently it looks like this:
```JAVA
final MessageEmbed failureEmbed = new EmbedBuilder().setAuthor("Error occured!", null, EmbedUtil.getJDAIconUrl())
    .setTitle(title, null)
    .setColor(Color.RED)
    .build();
reply(event, failureEmbed);
return;
```

```JAVA
// Return Info to User
final EmbedBuilder eb = new EmbedBuilder();
EmbedUtil.setColor(eb);

final MessageEmbed successEmbed = eb.setAuthor("Release Time of Version " + buildVersion, build.getUrl(), EmbedUtil.getJDAIconUrl())
    .setTitle(publishedTime, null)
    .build();

reply(event, successEmbed);
```

This seems better to me and as you said, easier to find. Normally I only use one embed if it is in the same scope but for this I can see why a second builder would be okay.

I guess since you marked this as resolved, you want it like it was originally with just one builder created in the beginning?

Reply:
I marked as resolved since you did have a point to use it like you did. i don't the way you do it. Both ways are fine by me